source/locale/pt_BR/LC_MESSAGES/installation-guide.po

```diff
@@ -104,66 +104,110 @@ msgstr ""
 msgid "Worker"
 msgstr ""
 
+#: ../../source/installation-guide.rst:39
+msgid "Balancer"
+msgstr ""
+
 #: ../../source/installation-guide.rst:40
+msgid "1 Core"
+msgstr "1 Núcleo"
+
+#: ../../source/installation-guide.rst:45
 msgid "In addition, you need:"
 msgstr "Além disso, são necessários:"
 
-#: ../../source/installation-guide.rst:42
+#: ../../source/installation-guide.rst:47
 msgid "**Network access**"
 msgstr "**Acesso à rede**"
 
-#: ../../source/installation-guide.rst:46
-msgid "The following ports should be opened:"
-msgstr "As seguintes portas devem estar abertas:"
+#: ../../source/installation-guide.rst:49
+msgid "The following ports should be opened for Docker Compose deployment:"
+msgstr "As seguintes portas devem estar abertas para a instalação com Docker Compose:"
 
-#: ../../source/installation-guide.rst:44
+#: ../../source/installation-guide.rst:50
 msgid ""
 "**TCP**: 8000 *(web interface access)*; 1883 *(MQTT, If you are going to "
-"use MQTT)*; 5896 *(LW2M, If you are going to use LW2M file server via "
+"use MQTT)*; 5896 *(LWM2M, If you are going to use LWM2M file server via "
 "HTTP instead of coap, UDP)*."
 msgstr ""
+"**TCP**: 8000 *(acesso a interface web)*; 1883 *(MQTT, se você irá usar MQTT)*; "
+"5896 *(LWM2M, se você irá usar o serviço de arquivos LWM2M por HTTP ao invés do "
```

Comment:
You will need to change the CoAP and LwM2M acronyms in this translation as well (in addition to the original in the file`.rst`)